akka-docs/rst/scala/stream/migration-guide-2.0-2.4-scala.rst

```diff
@@ -102,6 +102,19 @@ and were accessible via ``settings``. We now made it possible to configure the p
 settings as well, so ``RoutingSettings`` is now ``routingSettings`` and ``ParserSettings`` is
 now accessible via ``parserSettings``.
 
+Client / server behaviour on cancelled entity
+---------------------------------------------
+
+Previously if request received by server-side or response received by client-side were
```

Comment:
Maybe its enough to say: Previously if request or response were cancelled or consumer only partially...